pkg/minikube/machine/cache_images.go

```diff
@@ -106,24 +108,26 @@ func LoadImages(cc *config.MachineConfig, runner command.Runner, images []string
 		return errors.Wrap(err, "runtime")
 	}
 
+	cli, err := client.NewEnvClient()
+	if err != nil {
+		glog.Infof("error retrieving oci daemon client %v", err)
```

Comment:
Either log the error, or pass it up. Not both, in order to avoid duplicate error messages being logged.